### DISCUSSION TITLE
Does it matter which order I use the -shave and -resize operators in ?

### DISCUSSION THREAD

**Discussion by buforded** [ImageMagick/ImageMagick#5229] [Digital Image Processing]
I have a process where I *both* shave and resize an image:

`magick convert input.HEIC -quality 50 -shave 10x10 -resize 91% output.HEIC`

... what difference does it make if I change the order of 'shave' and 'resize' in a command like that ?

Thank you.

**Comment by snibgo** [comment]
If you swap the order, the number of pixels in the output will change. There will also be a slight performance loss, because "-resize" needs more CPU power than "-shave".
